src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -203,7 +203,7 @@ class BaseOptionsSelector extends Component {
         if (this.props.shouldFocusOnSelectRow) {
             // Input is permanently focused on native platforms, so we always highlight the text inside of it
             this.textInput.setNativeProps({selection: {start: 0, end: this.props.value.length}});
-            if (this.relatedTarget && ref === findNodeHandle(this.relatedTarget)) {
+            if (this.relatedTarget && ref === this.relatedTarget) {
```

Comment:
So it seems like the ref and relatedTarget are referring to the same instance of the internal object and thus check is passing. Cool!